test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -358,7 +358,7 @@ public async Task GetTrustResultAsync_RepositoryCountersignatureWithUntrustedRoo
             }
         }
 
-        [PlatformFact(Platform.Windows, Platform.Linux, Skip = "https://github.com/NuGet/Home/issues/12186")] // https://github.com/NuGet/Home/issues/11178
+        [PlatformFact(Platform.Windows, Platform.Linux)] // https://github.com/NuGet/Home/issues/11178
```

Comment:
Currently, signature verification on Mac is not supported, so we should probably close https://github.com/NuGet/Home/issues/11178 and remove the comments referencing it. See https://learn.microsoft.com/en-us/dotnet/core/tools/nuget-signed-package-verification

Reply:
If we ever enable signature verification on Mac, is it worth having that comment there?  Or will we just know to enable such tests at the same time?  I can remove the comment if so.

Reply:
I see you've already removed the comment. I don't understand how it helps us to remove it, since you point out, now whoever works on the test or related feature in the future won't know why only Windows platform is chosen.

Reply:
I'm not sure if we will plan to enable verification on macOS in the future. If we still need to, it might be good to keep the comments.